pkg/common/httpc.go

```diff
@@ -368,7 +368,7 @@ func download(targetUrl, storeFileName string, proxyConfig *ProxyConfig) bool {
 		_, err = io.Copy(out, resp.Body)
 		if err != nil {
 			fmt.Printf("Warning: cannot download file due to io error! error is %s\n", err.Error())
-			return false
+			//return false
```

Comment:
I think we should not forbid this false return for all errors, but need to give a err type check to see which kinds of err are  not real errors(like this "tls: user cancelled"), and bypass them.

Reply:
yeah, I agree.